### DISCUSSION TITLE
awesome-raylib resources

### DISCUSSION THREAD

**Discussion by uvtc** [raysan5/raylib#2378] [General]
Just learned of [awesome-raylib](https://github.com/Rabios/awesome-raylib). It looks ... pretty awesome.

Might be useful to have a link to this somewhere at https://www.raylib.com/ .

**Comment by archuser555** [comment]
Yeah i think you are right

**Comment by RobLoach** [comment]
Send a PR for the docs if you find it helpful 😉

**Comment by uvtc** [comment]
After looking at the raylib website, and at the github repo, and that repo's wiki, it seems like a good place for this is a new "Community Resources" page. I added it: https://github.com/raysan5/raylib/wiki/Community-Resources

And also added a link to this new wiki page in the wiki table of contents nav area (on the right at https://github.com/raysan5/raylib/wiki ).

**Comment by VoidStrikeX88** [comment]
the list is gone, any updates??

**Reply by RobLoach** [reply]
I think Rabios moved to gitlab or gitea.  Could be good to move that list from the Archive.org to the raylib wiki.

**Reply by raysan5** [reply]
note that list is quite outdated... like 2-4 years I think

**Reply by VoidStrikeX88** [reply]
I see... thanks for the heads up anyway, and for the "awesome" framework (see what i did there...)

**Reply by RobLoach** [reply]
I forked it, and cleaned it up a bit. Now living over in the raylib wiki at:
https://github.com/raysan5/raylib/wiki/Community-Resources
